From 3ac2b95c0a9d7de2be5c9f31ff0f9fbdd288345a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Miro=20Hron=C4=8Dok?= <miro@hroncok.cz>
Date: Mon, 10 Mar 2014 09:45:05 +0100
Subject: [PATCH] Make the desktop file valid

---
 icons/openscad.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/icons/openscad.desktop b/icons/openscad.desktop
index e43faea..4af6d18 100644
--- a/icons/openscad.desktop
+++ b/icons/openscad.desktop
@@ -4,5 +4,5 @@ Version=1.0
 Name=OpenSCAD
 Icon=openscad
 Exec=openscad %f
-MimeType=application/x-openscad
+MimeType=application/x-openscad;
 Categories=Graphics;3DGraphics;Engineering;Development;
-- 
1.8.5.5

